distributions/demo-google-deployment/transfer/build.gradle

```diff
@@ -23,16 +23,21 @@ import com.bmuschko.gradle.docker.tasks.image.Dockerfile
  * connects to a GCP project which it pulls app secrets from, etc, to test functionality of the
  * Google cloud extension without needing to deploy it to GCP.
  *
- * <p>TODO(#202): The next step will be to support deployed environments, and make sure the
- * resulting images work when deployed to GCP.
+ * <p>TODO(#202): Test deploying a prod image to GCP
  *
  * <p>TODO(@jimmarino): Automatically set cloudType=google so it is not needed on the commandline
  *
- * <ul>To run:
+ * <ul>To run locally:
  *  <li>./gradlew -PcloudType=google -PgcpProject=<your-project-name> \
- *      :distributions:demo-google-deployment:transfer:dockerize
+ *      :distributions:demo-google-deployment:transfer:dockerizeLocal
```

Comment:
Should we update this in the Documentation.md file as well?

Reply:
Where do you think it should go? Developer.md has instructions for demo-server but not demo-google-deployment currently. I'll add a comment about demo-google-deployment there.

Reply:
Oh youre right, I got confused with which distro was being run here. Maybe we should include this in the developer documentation under the google-demo-deployment (pointing to that in the developer.md)?  Maybe the README here: https://github.com/google/data-transfer-project/blob/master/distributions/demo-google-deployment/README.md ? Feel free to leave as is or create an issue to track the documentation changes we need.

Reply:
Good point, I"ll add something there once I get this working deployed to GCP, I'll document how to run it both locally and for prod (as part of #236)